provision/initramfs/capabilities/transport-http/wwgetfiles

```diff
@@ -54,9 +54,10 @@ elif [ -f "$TIMESTAMP_FILE" ]; then
 fi
 
 while true; do
+    echo "== wwgetfiles ==" >> /tmp/http.log
```

Comment:
It may be good to set this prior to the `while true; do` ... IIRC you would end up with multiple `== wwgetfiles ==` entries in the `http.log` file if the connection to the master timed out.

Reply:
You're right. Let me adjust that and force a push.